test/src/main/scala/scala/test/junit/JunitAbstractClassAndInterface.scala

```diff
@@ -0,0 +1,21 @@
+package scala.test.junit
+
+import org.junit.Test
+
+abstract class SomeAbstractTest {
+  @Test
+  def abstractTest: Unit =
+  	println("abstract")
+} 
+
+trait SomeTraitTest {
+  @Test
+  def traitTest: Unit =
+  	println("trait")
+} 
+
+class SingleTestSoTargetWillNotFailDueToNoTestsTest {
```

Comment:
And what about class that implements the trait or extends the abstract? Does it work the same?